### ISSUE TITLE
Don't build if no files changed (and same build command)

### ISSUE THREAD

**Issue by janpio** [ionic-team/ionic-cli#2820]
Right now Ionic CLI builds all the time. It would be nice if the CLI could skip a build if no files were changed and the build in `www` was done with the same params as the current command.

**Comment by dwieeb** [comment]
Yeah, definitely something I've been meaning to tackle in the CLI.
